### ISSUE TITLE
Jdk 16 builds failed on Linux and Windows

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/build-jdk#27]
jdk 16 linux, windows failed with message, three builds are affected. Suppose some new configuration setting has changed for jdk16, may from jdk15.

```
checking for dlopen in -ldl... yes
configure: error: Invalid value for --enable-jvm-feature-dtrace: 'auto'
configure exiting with result code 1
Error: No configurations found for /home/runner/work/build-jdk/build-jdk/openjdk-build/workspace/build/src.
Please run 'bash configure' to create a configuration.
```

**Comment by sophia-guo** [comment]
Failure is same as jdk15. #15

**Comment by adamfarley** [comment]
If this issue is talking about github action builds, then here's a related conversation (and potential solution):

https://adoptium.slack.com/archives/C09NW3L2J/p1620646105396700

TLDR: The dtrace "auto" setting appears to be hard-coded [here](https://github.com/adoptium/build-jdk/blob/12b8a9cb32d2c85448274ef26c81625a59797783/dist/index.js#L3357). Since "auto" is an unacceptable option for later versions of the JDK (jdk15 and up, I think), we should update the .js file to use "--enable-dtrace" instead of "--enable-dtrace=auto" for JDK15+ builds.

**Comment by sophia-guo** [comment]
Build looks good https://github.com/adoptium/build-jdk/actions/runs/836968650. Close by #31

**Comment by sophia-guo** [comment]
Looks like the failure wasn't fixed, https://github.com/sophia-guo/runaqaTest/runs/2693512311?check_suite_focus=true.

Nightly build succeed as https://github.com/adoptium/build-jdk/pull/31 passing in `skipFreetype` as string not a variable https://github.com/adoptium/build-jdk/blob/master/src/builder.ts#L99. Hence all arguments passing to makejdk-any-platform.sh was ignored and all builds are compiled as same. ( openj9 build is actually hotspot build)

```
/bin/bash ./makejdk-any-platform.sh -J /Users/runner/work/build-jdk/build-jdk/jdk/boot skipFreetype --configure-args --disable-warnings-as-errors --with-extra-cxxflags='-stdlib=libc++ -mmacosx-version-min=10.8' -d artifacts --target-file-name OpenJDK16U-jdk_x64_mac_hotspot_2021-05-27.tar.gz --use-jep319-certs --build-variant hotspot --disable-adopt-branch-safety jdk16u
Starting ./makejdk-any-platform.sh to configure, build (Adopt)OpenJDK binary
Parsing opt: -J
Possible opt arg: /Users/runner/work/build-jdk/build-jdk/jdk/boot
Working dir is ./build/
[debug] COPY_MACOSX_FREE_FONT_LIB_FOR_JDK_FLAG=false
[debug] COPY_MACOSX_FREE_FONT_LIB_FOR_JRE_FLAG=false
JDK Image folder name: jdk-bundle/jdk-*.jdk
JRE Image folder name: jre-bundle/jre-*.jre
Skipping setting boot JDK on docker host machine

```
https://github.com/adoptium/build-jdk/runs/2689267720?check_suite_focus=true

```
Run /Users/runner/work/build-jdk/build-jdk/openjdk-build/workspace/build/src/build/macosx-x86_64-server-release/jdk/bin/java -version
openjdk version "16.0.1" 2021-04-20
OpenJDK Runtime Environment Temurin-16.0.1+9-202105272310 (build 16.0.1+9-202105272310)
OpenJDK 64-Bit Server VM Temurin-16.0.1+9-202105272310 (build 16.0.1+9-202105272310, mixed mode)
```

**Comment by adamfarley** [comment]
PR raised to resolve: https://github.com/adoptium/build-jdk/pull/37

**Comment by adamfarley** [comment]
This problem doesn't occur anymore because we no longer attempt to enable dtrace on github action builds.

However, Jenkins pipeline builds still (afaik) include dtrace, and we still don't know *why* we can't include dtrace in github action builds, so it is still technically a problem.

Sophia, what do you think? Are green builds enough here, or is dtrace enablement worth further effort?
